rules/CodeQuality/Rector/If_/SimplifyIfElseToTernaryRector.php

```diff
@@ -211,6 +211,11 @@ private function haveNestedTernary(array $nodes): bool
 
     private function isNodeTooLong(Assign $assign): bool
     {
+        $originalAssign = $assign->getAttribute(AttributeKey::ORIGINAL_NODE);
+        if (! $originalAssign instanceof Assign) {
+            return true;
+        }
```

Comment:
check long assign print by original node first seems make error in test:

```diff
 1) Rector\Tests\CodeQuality\Rector\If_\SimplifyIfElseToTernaryRector\SimplifyIfElseToTernaryRectorTest::test with data set #0 (Symplify\SmartFileSystem\SmartFileInfo Object (...))
rules-tests/CodeQuality/Rector/If_/SimplifyIfElseToTernaryRector/Fixture/fixture.php.inc
Failed asserting that string matches format description.
--- Expected
+++ Actual
@@ @@
 {
     public function run()
     {
-        $this->arrayBuilt[][$key] = empty($value) ? true : $value;
+        if (empty($value)) {
+            $this->arrayBuilt[][$key] = true;
+        } else {
+            $this->arrayBuilt[][$key] = $value;
+        }
     }
 }
```
I will check.